controllers/disruption_controller.go

```diff
@@ -202,13 +194,17 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		if err != nil {
 			return ctrl.Result{}, fmt.Errorf("error updating disruption injection status: %w", err)
 		} else if !injected {
-			r.log.Infow("disruption is not fully injected yet, requeuing")
+			requeueAfter := time.Duration(rand.Intn(5)+5) * time.Second //nolint:gosec
```

Comment:
Can you add a small comment about that line just to specify that it requeues for 5-10s please? I guess re-reading that line later can be a bit painful for nothing.